pytest.ini

```diff
@@ -10,6 +10,8 @@ addopts =
     --ignore=baybe/_optional
     --ignore=baybe/utils/chemistry.py
     --ignore=tests/simulate_telemetry.py
+    --ignore=baybe/utils/diagnostics.py
+    --ignore=tests/utils/test_diagnostics.py
```

Comment:
This was an (arguably bad) implementation to prevent errors when coretests are performed. I have now replaced this with a `pytest.mark.skipif` statement in the related test. Similarly, I added a conditional to the new `shap_explainers` and `non_shap_explainers` fixtures to ensure no errors are raised when the `shap` package is not installed.